.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Feature Request
+about: Suggest an idea for the roadmap! The team will review
+title: "[request]: describe request here"
+labels: Proposed
```

Comment:
consider `feature-request`

Reply:
I think if we broaden the template name to "Request", `feature-request` as an auto-tag doesn't make as much sense. I'll change this to be `Request`

Reply:
Done